romancal/datamodels/dynamicdq.py

```diff
@@ -10,26 +10,40 @@
 
 """
 
-from .dqflags import pixel, group
-from stcal import dynamicdq
-
-def dynamic_mask(input_model, mnemonic_map=pixel):
-    """
-    Return a mask model given a mask with dynamic DQ flags.
-
-    Dynamic flags define what each plane refers to using the DQ_DEF extension.
-
-    Parameters
-    ----------
-    input_model : ``MaskModel``
-        An instance of a Mask model defined in jwst or romancal.
-    mnemonic_map : dict
-
-    Returns
-    -------
-    dqmask : ndarray
-        A Numpy array
-    """
-
-    return dynamicdq.dynamic_mask(input_model, mnemonic_map)
-
+import logging
```

Comment:
Perhaps comments in this file can clarify what this code does, since dynamic dq has been removed.

Reply:
This file was removed from the PR.